packages/mwp-app-render/src/components/StyleWith.jsx

```diff
@@ -28,7 +28,9 @@ const StyleWith = props => {
 
 	const moduleCSS = new Set();
 	styles.forEach(style => {
-		moduleCSS.add(style._getCss()); // eslint-disable-line no-underscore-dangle
+		if (typeof style._getCss === 'function') {
```

Comment:
might be worth a code comment to explain when this would _not_ be true. Also, is there a way to add a mock somewhere that would set this up automatically in tests? I can't remember the details of what how the loader works, but if we could use `moduleNameMapper` config in Jest to get this to work, that would be preferable.

Reply:
Yeah, this is better handled by mocks for `module.scss` and `StyleWith` imports. Closing this PR.